packages/gestalt-design-tokens/build.js

```diff
@@ -771,6 +787,7 @@ StyleDictionary.registerTransformGroup({
     'name/cti/snake',
     'name/conflictFixing',
     'color/hex8android',
```

Comment:
Be careful , in Android we have `sp`
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/ad1b37bc-7cfd-4337-adc6-1f40d7d5ee4c)

Reply:
iOS uses px.

Reply:
Thanks! Yup this converts to px first.

All snapshots pass with the transformer